docs/items/weapon-configs.md

```diff
@@ -142,4 +142,450 @@ What values a Weapon Type can have:
   * 14+42
 * REQ Tier: 3
 * Game State: 2
-* Notes: -
\ No newline at end of file
+* Notes: -
+  
+## [11] Boxer
+* Weapon Type: CQS48 Bulldog + BR75 Breacher
+* Trait Set: 11
+  * Weapon Damage: 1.0
+* Config: Combo
+* Ammo: Default
+  * 30+90
+* REQ Tier: 4
+* Game State: 3
+* Notes:-
+  
+## [12] Blaze Of Glory
+* Weapon Type: CQS48 Bulldog + Diminisher Of Hope
+* Trait Set: 12
+  * Weapon Damage: 2.90
+* Config: Combo
+* Ammo: Default
+  * 7+7
+* REQ Tier: 5
+* Game State: 4
+* Notes:-
+  
+## [13] Nornfang
+* Weapon Type: S7 Sniper + Diminisher Of Hope
+* Trait Set: 13
+  * Weapon Damage: 1.61
+* Config: Custom
```

Comment:
Nornfang is a Projectile config weapon and needs more info on what projectile is used and with what settings. Please use the config template provided in this same file starting from line 9 to write down this info when making a config.